### ISSUE TITLE
Tags for wcag21a and wcag21aa are missing from the rules.

### ISSUE THREAD

**Issue by nurulquamar** [dequelabs/axe-core#3760]
### Product

axe-core

### Latest Version

- [X] I have tested the issue with the latest version of the product

#### Expectation
I should be able to detect the issue w.r.t the WCAG 2.1 A and WCAG 2.1 AA guidelines.

#### Actual
I am able to find issues w.r.t wcag2a and wcag21aa but not for wcag21a and wcag21aa.

#### How to Reproduce
Set the rules to wcag21a or wcag21aa and run the tests against any webpage. You will not be able to detect the violations which are part of only wcag21a or wcag21aa.

#### Additional context
The rules in the axe.js are not updated with the tags for wcag21a and wcag21aa. For example, for the rule duplicate-id-active, the code block is as follows:
```
{
      id: 'duplicate-id-active',
      selector: '[id]',
      matches: 'duplicate-id-active-matches',
      excludeHidden: false,
      tags: [ 'cat.parsing', 'wcag2a', 'wcag411' ],
      all: [],
      any: [ 'duplicate-id-active' ],
      none: []
    }

```
You can see that the tags include wcag2a but not wcag21a. Whereas when you see the violation on the deque website: https://dequeuniversity.com/rules/axe/4.4/duplicate-id-active, you notice that it applies to both WCAG 2.0 A and WCAG 2.1 A.
A similar issue was raised in Cypress axe, but I think it has more to do with axe-core: https://github.com/component-driven/cypress-axe/issues/123

**Comment by dylanb** [comment]
The rule tests specifically a S.C. that is part of WCAG 2 level A i.e. 4.1.1

It is clear that WCAG includes all S.C. that are part of WCAG 2.0 but the tags specify precisely which S.C. any rule applies to
